kolibri/plugins/coach/assets/src/composables/useQuizCreation.js

```diff
@@ -422,10 +421,13 @@ export default function useQuizCreation(DEBUG = false) {
   );
   /** @type {ComputedRef<QuizExercise[]>}   The active section's `resource_pool` */
   const activeResourcePool = computed(() => get(activeSection).resource_pool);
-  /** @type {ComputedRef<ExerciseResource[]>} The active section's `resource_pool` - that is,
-   *                                          Exercises from which we will enumerate all
-   *                                          available questions */
-  const activeExercisePool = computed(() => get(activeResourcePool).filter(isExercise));
+  /** @type {ComputedRef<QuizExercise[]>}   The active section's `resource_pool` */
+  const activeResourceMap = computed(() =>
```

Comment:
The alternative here would be to store an activeResourceMap as the basic representation, and have `Object.values` for `activeExercisePool` if it is ever needed as an array.

Looks like this is never exported, so maybe we should just set it as a map in the first place?